hugegraph-test/src/main/java/com/baidu/hugegraph/api/MetricsApiTest.java

```diff
@@ -100,61 +100,62 @@ public void testMetricsBackend() {
-//                    value = e.getValue();
-//                    Assert.assertTrue(String.format(
-//                                      "Expect map value for key %s but got %s",
-//                                      key, value),
-//                                      value instanceof Map);
-//                    host = (Map<?, ?>) value;
-//                    assertMapContains(host, "mem_max");
-//                    assertMapContains(host, "mem_committed");
-//                    assertMapContains(host, "mem_used");
-//                    assertMapContains(host, "mem_used_readable");
-//                    assertMapContains(host, "mem_unit");
-//
-//                    assertMapContains(host, "disk_usage");
-//                    assertMapContains(host, "disk_usage_readable");
-//                    assertMapContains(host, "disk_usage_details");
-//                    assertMapContains(host, "disk_unit");
-//
-//                    assertMapContains(host, "uptime");
-//                    assertMapContains(host, "uptime_readable");
-//                    assertMapContains(host, "time_unit");
-//
-//                    assertMapContains(host, "estimated_partition_count");
-//                    assertMapContains(host, "dropped_mutations");
-//                    assertMapContains(host, "pending_flushes");
-//                    assertMapContains(host, "key_cache_hit_rate");
-//                    assertMapContains(host, "bloom_filter_false_ratio");
-//
-//                    assertMapContains(host, "write_latency_hugegraph");
-//                    assertMapContains(host, "read_latency_hugegraph");
-//                    assertMapContains(host, "write_latency_*");
-//                    assertMapContains(host, "read_latency_*");
-//
-//                    assertMapContains(host, "key_cache_size");
-//                    assertMapContains(host, "key_cache_entries");
-//                    assertMapContains(host, "row_cache_size");
-//                    assertMapContains(host, "row_cache_entries");
-//                    assertMapContains(host, "counter_cache_size");
-//                    assertMapContains(host, "counter_cache_entries");
-//
-//                    assertMapContains(host, "compaction_completed_tasks");
-//                    assertMapContains(host, "compaction_pending_tasks");
-//                    assertMapContains(host, "compaction_bytes_compacted");
-//
-//                    assertMapContains(host, "live_nodes");
-//                    assertMapContains(host, "joining_nodes");
-//                    assertMapContains(host, "moving_nodes");
-//                    assertMapContains(host, "leaving_nodes");
-//                    assertMapContains(host, "unreachable_nodes");
-//
-//                    assertMapContains(host, "keyspaces");
-//                    assertMapContains(host, "num_tables");
-//                    assertMapContains(host, "exception_count");
-//                }
+
```

Comment:
we can rebase on master branch https://github.com/apache/incubator-hugegraph/pull/1853

Reply:
ok